### DISCUSSION TITLE
关于文字检测和文字识别的疑惑

### DISCUSSION THREAD

**Discussion by pcycccccc** [PaddlePaddle/PaddleOCR#13025] [Q&A] [answered]
您好，我对PaddleOCR的模型训练有点不大明白，想跟您请教一下。我的项目任务是对图片上的文字内容进行识别（我的图片里，有一部分是有文字的，我并不在乎模型输出文字的位置给我，我只需要文字的内容），然后我注意到了PaddleOCR这个项目，我看到它里边分了几个任务，一个是文字检测，一个是文字识别，其他的我没有注意了。我按照我的理解，我的项目任务应该只需要文字识别就可以了吧，那我是不是只需要训练文字识别的模型就可以，用训练好的模型进行下一步推理？但是我注意到官方提供的Windows cpp部署教程里，输入的参数除了文字识别模型还有一个文字检测模型，这个文字检测模型是必须需要训练的吗？
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/108455796/ce2ec3cb-2cc6-4d73-a4ef-906f95267de4)

有人能帮我回答一下问题吗？

**Comment by TOMORINAONAO** [comment, marked as answer]
文字识别模型只能识别单行文字，如果你的图片有多行文字，需要先用文字检测模型将图片中的所有文本行检测出来，再给识别模型一行行识别。另外，paddleocr有现成训练好的检测模型和识别模型，检测模型大部分情况下都不需要自己训练用现成的就行，识别模型识别一些生僻字效果不好的可能需要自己训练

**Comment by GreatV** [comment]
你可以先试试官方提供的通用模型，看看是文字识别得不准还是检测不到文字的位置，然后再针对性的训练。

**Comment by pcycccccc** [comment]
@GreatV @TOMORINAONAO  嘻嘻，你们真好！我明白你们的意思了！！我会去试试的！
